arcgis-ios-sdk-samples/Search/Find place/FindPlaceViewController.swift

```diff
@@ -134,11 +134,11 @@ class FindPlaceViewController: UIViewController, UITableViewDataSource, UITableV
     }
     
     //method to toggle the suggestions table view on and off
-    private func animateTableView(expand:Bool) {
+    fileprivate func animateTableView(_ expand:Bool) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage